fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -1259,7 +1260,6 @@ nonReserved
     | NON_NULLABLE
     | NULLS
     | OF
-    | OFFSET
```

Comment:
i don't think add reserved keyword to support non-mysql dialect is good idea. because doris use mysql protocol, so we should make doris as mysql like as possible. if want migrate from presto, use [doris sql convertor](https://doris.apache.org/zh-CN/docs/lakehouse/sql-dialect/) is a better way

Reply:
https://dev.mysql.com/doc/refman/8.4/en/keywords.html actually it is keyword in mysql